tests/run-make/type-mismatch-sysroot-crate/hashbrown.rs

```diff
@@ -0,0 +1,8 @@
+use std::marker::PhantomData;
+
+pub struct HashMap<T, U>(PhantomData<(T, U)>);
```

Comment:
why the phantomdata and default instead of just making it a unit struct?

Reply:
huh, didn't expect that to work. ty

Reply:
your generics dont match hashbrown anyways, it has 4 :3